src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingService.java

```diff
@@ -240,8 +241,8 @@ private ListingData getDateRangeReport(ListingDetails listingDetails, String aut
     private List<SubmitEvent> getDateRangeReportSearch(ListingDetails listingDetails, String authToken)
             throws IOException {
         var listingData = listingDetails.getCaseData();
-        boolean dateRange = listingData.getHearingDateType().equals(RANGE_HEARING_DATE_TYPE);
-        if (!dateRange) {
+        boolean isRangeHearingDateType = listingData.getHearingDateType().equals(RANGE_HEARING_DATE_TYPE);
```

Comment:
I think most people tend to use the is for the method name rather than the variable